proposals/0000-rich-errors-proposal.rst

```diff
@@ -255,6 +304,33 @@ be useful. We hope that the discussion that arises from this proposal will shed
 light on additional items. Moreover, we anticipate that the vocabulary will
 grow in time as new tooling applications are found.
 
+A smaller but very concrete challenge is figuring out how to give users a hook
+into the processing of error documents when they're reported (and possibly
+in other places where our prototype implementation had to "strip off"
+annotations) because of the lack of such a hook. Our prototype just applies
```

Comment:
The "Our prototype..." sentence either needs to be made more concrete or more abstract. Currently it's a bit unclear what is meant.

Reply:
Updated in latest revision.